src/model/FaceliftEnum.cpp

```diff
@@ -32,10 +32,15 @@
 #include <QDebug>
 
 namespace facelift {
+    
+namespace Enum {
```

Comment:
lowercase namespace please. This will allow us to distinguish it from classes

Reply:
lowercase name "enum" is reserved keyword.
Other names would be worst...

Reply:
let it be Enum as exception